src/main/java/de/tum/in/www1/artemis/repository/ExamRepository.java

```diff
@@ -243,7 +244,6 @@ AND TYPE(ex) = QuizExercise
             FROM Exam e
                 LEFT JOIN FETCH e.exerciseGroups eg
                 LEFT JOIN FETCH eg.exercises ex
-            WHERE TYPE(ex) = QuizExercise
```

Comment:
Are you sure this is correct? Before only quizExercises were included, now we'd load all exercises (which probably already exists as a different query method).

Edit: Looking at the code, this gets used in a course context (`courses/{courseId}/exams-for-user`) but does not filter by course id here. We should look into this in more detail in a followup.

Reply:
No, failed revert.